scss/_close.scss

```diff
@@ -4,25 +4,39 @@
 // See https://developer.mozilla.org/en-US/docs/Web/Events/click#Safari_Mobile
 
 .btn-close {
-  padding: $btn-close-padding;
-  color: $btn-close-color;
-  background-color: transparent; // include transparent for button elements
-  border: $btn-border-width solid transparent; // Boosted mod
+  // scss-docs-start btn-close-css-vars
+  --#{$prefix}btn-close-padding: #{$btn-close-padding}; // Boosted mod
+  --#{$prefix}btn-close-margin: 0; // Boosted mod
+  --#{$prefix}btn-close-border-width: #{$btn-close-border-width}; // Boosted mod
+  --#{$prefix}btn-close-color: #{$btn-close-color}; // Boosted mod
+  --#{$prefix}btn-close-bg: #{$btn-close-bg}; // Boosted mod
+  --#{$prefix}btn-close-border-color: #{$btn-close-border-color}; // Boosted mod
+  --#{$prefix}btn-close-hover-color: #{$btn-close-hover-color}; // Boosted mod
+  --#{$prefix}btn-close-active-color: #{$btn-close-active-color}; // Boosted mod
+  --#{$prefix}btn-close-active-border-color: #{$btn-close-active-border-color}; // Boosted mod
+  --#{$prefix}btn-close-disabled-color: #{$btn-close-disabled-color}; // Boosted mod
+  // scss-docs-end btn-close-css-vars
+
+  padding: var(--#{$prefix}btn-close-padding); // Boosted mod
+  margin: var(--#{$prefix}btn-close-margin); // Boosted mod
```

Comment:
This extra-rule didn't exist before. I'm not sure it should be added since it doesn't bring anything here.